<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistChangeWidget</class>
 <widget class="QWidget" name="PlaylistChangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="LFOWidget" name="lfoWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="SSGNoiseFreqWidget" name="ssgNoiseFreqWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="SSGEnvelopeFreqWidget" name="ssgEnvFreqWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="SSGEnvelopeShapeWidget" name="ssgEnvShapeWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="FM2Widget" name="userToneWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>224</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="doneButton">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SSGEnvelopeShapeWidget</class>
   <extends>QWidget</extends>
   <header>ssg/ssgenvelopeshapewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SSGEnvelopeFreqWidget</class>
   <extends>QWidget</extends>
   <header>ssg/ssgenvelopefreqwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SSGNoiseFreqWidget</class>
   <extends>QWidget</extends>
   <header>ssg/ssgnoisefreqwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LFOWidget</class>
   <extends>QWidget</extends>
   <header>playlistwidget/lfowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FM2Widget</class>
   <extends>QWidget</extends>
   <header>usertone/fm2widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
